konan_sdk/sdk.py

```diff
@@ -39,6 +50,37 @@ def predict(self, deployment_uuid: str, input_data: Union[Dict, str]) -> Tuple[s
         # Check if access token is valid and retrieve a new one if needed
         self.auth.auto_refresh_token()
 
-        prediction_uuid, output = PredictionEndpoint(api_url=self.api_url, user=self.auth.user, deployment_uuid=deployment_uuid).post(payload=input_data)
+        prediction = PredictionEndpoint(
+            self.api_url,
+            deployment_uuid=deployment_uuid, user=self.auth.user
+        ).post(input_data)
+        return prediction.uuid, prediction.output
 
-        return prediction_uuid, output
+    def evaluate(
+        self, deployment_uuid: str,
+        start_time: datetime.datetime, end_time: datetime.datetime
+    ) -> List[KonanBaseMetric]:
+        """Call the evaluate function for a given deployment
+
+        :param deployment_uuid: uuid of deployment to use for evaluation
+        :type deployment_uuid: str
+        :param start_time: use predictins made at or after this time
+        :type start_time: datetime.datetime
+        :param end_time: use predictins made before or at this time
+        :type end_time: datetime.datetim
+        :return: A model evaluation object
+        :rtype: EvaluateEndpoint.ResponseObject
```

Comment:
minor typo in `end_time: datetime.datetim` and return type needs to be updated

Reply:
Fixed